internal/conf/configuration.go

```diff
@@ -421,16 +422,69 @@ func (c *CaptchaConfiguration) Validate() error {
 	return nil
 }
 
+type DatabaseEncryptionConfiguration struct {
+	Encrypt bool `json:"encrypt"`
```

Comment:
what about calling this `Enabled` instead - so `GOTRUE_SECURITY_DB_ENCRYPTION_ENABLED`

Reply:
It's not about enabling, it's about encrypting. Once you encrypt even one row, you can't "turn it off." You can turn off encrypting new rows -- that's why it's called `Encrypt` -- but you must always be able to have access to the decryption keys, which is the other field below with the map.

Reply:
ah ok would be great to add a comment to explain this nuance